frontend/src/metabase/query_builder/components/QueryModals.tsx

```diff
@@ -232,7 +232,7 @@ class QueryModals extends Component<QueryModalsProps> {
         );
       case MODAL_TYPES.FILTERS:
         return (
-          <Modal fit onClose={onCloseModal}>
+          <Modal fit onClose={onCloseModal} trapFocus={false}>
```

Comment:
For some reason, in the `filter-bulk` tests, the focus trap causes the tests to fail because keyboard shortcuts stop working (the FocusTrap prevents them from happening). So, for now, I added this prop to handle this - and thankfully users can't keyboard navigate out of this modal anyway.  In the future, when we eventually move to Mantine modals, we won't have to worry about choosing between focus traps and keyboard shortcuts.